### PR TITLE
search frontend: render filter bars even when empty

### LINKED ISSUE #17560: Streaming: jittery results display on long-running searches
On a long-running streaming search like [`RouteHandler timeout:20s count:1000`](https://sourcegraph.com/search?q=RouteHandler+timeout:20s+count:1000&patternType=literal), notice that the results are pushed down three times:

1. When the filters row is rendered
2. When the "skipped" toolbar item is rendered
3. When the repositories row is rendered

Item 2. would not happen on wider viewports, but I find the unstable UI with the actual results being pushed further and further below the fold frustrating (especially if I start reading the code of a result while the search is still running, and it suddenly skips down). Maybe something we only want to tackle with the results page refactor, though?

cc @rrhyne @limitedmage 

https://user-images.githubusercontent.com/1741180/105499822-4ff81000-5cc2-11eb-9a0a-5cfecb5fbbb6.mp4

### PR DESCRIPTION
**Pull request by limitedmage**
Fixes #17560

PS. Review with "Hide whitespace changes" turned on

image

### CODE REVIEW

**Comment by limitedmage** [comment]
Chromatic seems to be rendering the scroll buttons on the empty bars in their snapshots, this does NOT happen in either the real product or the local storybook :S

image

**Comment by eseliger** [comment]
oh no 🤦

File: client/web/src/search/results/SearchResultsFilterBars.scss [hunk 1]
```diff
@@ -37,6 +37,14 @@
         border-bottom: 1px solid var(--border-color);
     }
 
+    &__row-label {
+        margin: 0.25rem 0;
```
Comment (eseliger) [inline client/web/src/search/results/SearchResultsFilterBars.scss:41]: isn't that `.my-1`?

**Comment by limitedmage** [comment]
> oh no 🤦

Is this about the change itself or the Chromatic issue? :P

**Review by eseliger — APPROVED** [review]

**Comment by eseliger** [comment]
> Is this about the change itself or the Chromatic issue? :P

Chromatic 😬 this PR is "oh yes"

**Comment by rrhyne** [comment]
Looks great live!

**Pull request merged**